tests/test_bump_create_tag.py

```diff
@@ -13,9 +14,13 @@
     (("1.2.3+1.0.0", "v$version"), "v1.2.3+1.0.0"),
     (("1.2.3+1.0.0", "v$version-local"), "v1.2.3+1.0.0-local"),
     (("1.2.3+1.0.0", "ver$major.$minor.$patch"), "ver1.2.3"),
+    (("1.2.3a1", "%y.%m.%d$prerelease"), "21.05.06a1"),
+    (("1.2.3a1", "%Y.%-m.%-d$prerelease"), "2021.5.6a1"),
+    (("1.2.3a1", "%y.%-m.$major.$minor"), "21.5.1.2"),
```

Comment:
What happens when going from calver to calver?

Reply:
This currently fails because create_tag expects version.release to be three integers (in Version, splits on '.' and maps to int) so a CalVer of `21.5.1.2` would fail here:

https://github.com/commitizen-tools/commitizen/blob/bec20ebf433f2281c70f1eb4b0b6a1d0ed83e9b2/commitizen/bump.py#L226

This will definitely need to be fixed. I'll check how custom formats (dashes, etc.) are handled by commitizen and give this more thought